earthpy/spatial.py

```diff
@@ -369,13 +369,13 @@ def crop_image(raster, geoms, all_touched=True):
         >>> from earthpy.io import path_to_example
         >>> # Clip an RGB image to the extent of Rocky Mountain National Park
         >>> rmnp = gpd.read_file(path_to_example("rmnp.shp"))
-        >>> with rio.open(path_to_example("rmnp-rgb.tif")) as raster:
-        ...     src_image = raster.read()
-        ...     out_image, out_meta = es.crop_image(raster, rmnp)
+        >>> with rio.open(path_to_example("rmnp-rgb.tif")) as src:
+        ...     in_image = src.read()
```

Comment:
```suggestion
```
Let's remove this line altogether because what happens is students get confused and think to crop you need to both read and crop and that is not true!!